srcjs/notifications.js

```diff
@@ -0,0 +1,160 @@
+exports.notifications = (function() {
+
+  // Milliseconds to fade in or out
+  const fadeDuration = 250;
+
+  function show({ html=null, duration=5000, id=null, closeButton=true,
+                  style = null } = {})
```

Comment:
I like `type` because I think it's how you'd naturally think about this -- as in "what's the type of this notification?" and the answer (info, warning, ...) makes sense.